grobid-trainer/resources/dataset/citation/corpus/crossref_raw_citations.xml

```diff
@@ -0,0 +1,54 @@
+<bibl> <author>Meng Qu and Jian Tang</author>. <date>2019</date>. <title level="a">Probabilistic Logic Neural Networks for Reasoning</title>. <idno type="arXiv">arxiv:cs.LG/1906.08495</idno> </bibl>
+<bibl> <author>Bishan Yang Scott Wen-tau Yih Xiaodong He Jianfeng Gao and Li Deng</author>. <date>2015</date>. <title level="a">Embedding Entities and Relations for Learning and Inference in Knowledge Bases</title>. In <title level="m">Proceedings of the International Conference on Learning Representations (ICLR)</title> <date>2015</date>. <ptr type="web">https://www.microsoft.com/en-us/research/publication/embedding-entities-and-relations-for-learning-and-inference-in-knowledge-bases/</ptr> </bibl>
+<bibl> <author>Christoph Molnar</author>. <date>2020</date>. <title level="m">Interpretable Machine Learning</title> .<publisher>Leanpub</publisher> <pubPlace>Victoria Canada</pubPlace>. <ptr type="web">https://christophm.github.io/interpretable-ml-book/</ptr>. </bibl>
+<bibl> <author>Günter Klambauer Thomas Unterthiner Andreas Mayr and Sepp Hochreiter</author>. <date>2017</date>. <title level="a">Self-Normalizing Neural Networks</title>. <idno type="arXiv">arxiv: 1706.02515</idno> [cs.LG] </bibl>
+<bibl> <author>Byung-Hak Kim Ethan Vizitei and Varun Ganapathi</author>. <date>2018</date>. <title level="a">GritNet: Student Performance Prediction with Deep Learning</title>. <idno type="arXiv">arxiv: 1804.07405</idno> [cs.LG] </bibl>
+<bibl> <author>Sercan O. Arik and Tomas Pfister</author>. <date>2019</date>. <title level="a">TabNet: Attentive Interpretable Tabular Learning</title>. <idno type="arXiv">arxiv: 1908.07442</idno> [cs.LG] </bibl>
+<bibl> <orgName>Wikipedia</orgName>. <date>2019</date>. <title level="m">PageRank</title>, Retrieved <date>September 5, 2019</date>, from <ptr type="web">https://en.wikipedia.org/wiki/PageRank</ptr> </bibl>
+<bibl> <author>Mohammad Al Hasan, Vineet Chaoji, Saeed Salem, and Mohammed Zaki</author>. [n.d.]. <title level="m">Link Prediction using Supervised Learning</title>. ([n. d.]), 10. </bibl>
+<bibl> <author>Johan J Vossensteyn Andrea Kottmann Benjamin WA Jongbloed Franciscus Kaiser Leon Cremonini Bjorn Stensaker Elisabeth Hovdhaugen and Sabine Wollscheid</author>. <date>2015</date>. <title level="m">Dropout and completion in higher education in Europe: Main report</title>. <note type="report">Technical Report</note>. <orgName>CHEPS</orgName> and <orgName>NIFU</orgName> . </bibl>
+<bibl> <author>Xue Bin Peng, Marcin Andrychowicz, Wojciech Zaremba, and Pieter Abbeel</author>. <date>2018</date>. <title level="a">Sim-to-Real Transfer of Robotic Control with Dynamics Randomization</title>. <title leve="m">Proceedings - IEEE International Conference on Robotics and Automation</title> (<date>2018</date>), <biblStruct type="pp">3803--3810</biblStruct>. </bibl>
```

Comment:
`<biblStruct type="pp">3803--3810</biblStruct>` -> `<biblScope unit="page">3803--3810</biblScope>`

Note: in the latest TEI guidelines, we have `unit="page"` instead of `type="pp"` (both TEI compliant), but the Grobid training data parser covers both.